src/api/store/event.py

```diff
@@ -2,14 +2,62 @@
 from _main_.utils.massenergize_errors import MassEnergizeAPIError, InvalidResourceError, ServerError, CustomMassenergizeError, NotAuthorizedError
 from _main_.utils.massenergize_response import MassenergizeResponse
 from django.db.models import Q
+from django.utils import timezone
 from _main_.utils.context import Context
 from sentry_sdk import capture_message
 from .utils import get_user_or_die
+#from _main_.utils.common import parse_date
 import datetime
 from datetime import timedelta
 import calendar
 import pytz
 
+def _check_recurring_date(start_date_and_time, end_date_and_time, day_of_week, week_of_month):
```

Comment:
this is the kind of thing i would loooooove to see some tests for.  What do you think about adding some good ol tests to capture this?

Reply:
Isha added some tests